components/constellation/constellation.rs

```diff
@@ -1572,6 +1573,65 @@ where
             EmbedderToConstellationMessage::EmbedderControlResponse(id, response) => {
                 self.handle_embedder_control_response(id, response);
             },
+
+            EmbedderToConstellationMessage::CreateEntangledMessagePorts(
+                router_id,
+                sender,
+                port1,
+                port2,
+            ) => {
+                trace!("creating new entangled ports {port1:?} {port2:?}");
+                self.handle_new_messageport_router(router_id, sender);
+                self.handle_new_messageport(router_id, port1);
+                self.handle_new_messageport(router_id, port2);
+                self.handle_entangle_messageports(port1, port2);
```

Comment:
Same here, the embedder can use the existing api and send several messages, as opposed to add a new api.